plugin.audio.euskarazko-irratiak/resources/lib/arrosa_scraper.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+#    Copyright (C) 2016 Asier Iturralde Sarasola
+#
+#    This file is part of plugin.audio.euskarazko-irratiak.
+#
+#    plugin.audio.euskarazko-irratiak is free software: you can redistribute it
+#    and/or modify it under the terms of the GNU General Public License as
+#    published by the Free Software Foundation, either version 3 of the License,
+#    or (at your option) any later version.
+#
+#    plugin.audio.euskarazko-irratiak is distributed in the hope that it will
+#    be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with plugin.audio.euskarazko-irratiak.
+#    If not, see <http://www.gnu.org/licenses/>.
+#
+
+# http://docs.python-requests.org/en/latest/
+import requests
+from bs4 import BeautifulSoup
+
+ARROSA_PODCASTS_URL = 'http://www.arrosasarea.eus/category/irratien-programak/'
+
+def format_date(date):
+    months = ["urtarrila", "otsaila", "martxoa", "apirila", "maiatza", "ekaina", "uztaila", "abuztua", "iraila", "urria", "azaroa", "abendua"]
```

Comment:
The names of the months are not part of the user interface. I think that it doesn't make sense to translate them. I use the format_date function to change the format of the dates of the audios that I scrape from a website. For example, the date of the first audio from [this page](http://www.arrosasarea.eus/category/irratien-programak/97-irratia/) is "abuztua 25, 2016", but I want to display it in the list as "2016/08/25", that's what the format_date function does.
Thanks for reviewing my plugin!